index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=index, initial-scale=1.0" />
+    <title>index</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Roboto"
+    />
+  </head>
+  <body>
```

Comment:
Use semantics like header and main to structure your code in a more descriptive way

Reply:
Please make sure to use a polite way of suggesting a change instead of ordering. So that the author won't take it wrong and will love to implement the changes or discuss his/her reasons with you.